lib/lita/handler/chat_router.rb

```diff
@@ -86,10 +86,8 @@ def dispatch_to_route(route, robot, message)
         robot.hooks[:trigger_route].each { |hook| hook.call(response: response, route: route) }
         handler = new(robot)
         route.callback.call(handler, response)
-      rescue Exception => e
-        log_dispatch_error(e)
-        robot.config.robot.error_handler.call(e)
-        raise e if Lita.test_mode?
+      rescue => error
```

Comment:
I took the liberty to rescue `StandardException` instead. `Exception` can catch things you don't want to ignore.